### PR TITLE
Implemented generator

### PR DESCRIPTION
**Pull request by rocket111185**
* Implemented CSV generator
* Generated CSV file, lol

### CODE REVIEW

File: generator/main.js [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+  faker.date.past(),
+];
 
-console.log(generateProduct()); // TODO: write to file
+const arrayToRow = (array) => {
+  const enquotedArray = array.map((el) => `"${el}"`);
+  return enquotedArray.join(',') + '\n';
+};
+
+for (let i = 0; i < 1000; i++) {
```
Comment (yehorbk) [inline generator/main.js]: Move 1000 to a const variable at the top of the file

File: generator/main.js [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+  const product = generateProduct();
+  const row = arrayToRow(product);
+  appendFileSync('contents.csv', row);
```
Comment (yehorbk) [inline generator/main.js]: You should put the file into `data` directory and call it `products.csv`. The `data` directory is in the root directory, path from generator is `../data`.

File: generator/main.js (hunk 1 above)
Comment (rocket111185) [reply generator/main.js]: Fixed in the recent commit

File: generator/main.js (hunk 2 above)
Comment (rocket111185) [reply generator/main.js]: Fixed in the recent commit

**Comment by yehorbk** [comment]
Perfect!

**Pull request merged**